apps/sms/lib/sww.js

```diff
@@ -25,16 +25,19 @@ Router.prototype.methods = ['get', 'post', 'put', 'delete', 'head',
  * @param handler (Function) payload to be executed if url matches.
```

Comment:
I suppose this file won't be included in the patch, but leave it here just as a reminder :)